perfkitbenchmarker/linux_benchmarks/beam_integration_benchmark.py

```diff
@@ -67,31 +77,46 @@ def GetConfig(user_config):
   return configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
 
 
-def CheckPrerequisites(benchmark_spec):
+def CheckPrerequisites(benchmark_config_spec):
   """Verifies that the required resources are present.
 
   Raises:
     perfkitbenchmarker.errors.Config.InvalidValue: If no Beam args are provided.
     NotImplementedError: If an invalid runner is specified.
   """
-  if FLAGS.beam_it_args is None:
+  if FLAGS.beam_it_options is None and FLAGS.beam_it_args is None:
     raise errors.Config.InvalidValue(
-        'No args provided. To run with default class (WordCountIT), must'
-        'provide --beam_it_args=--tempRoot=<temp dir, e.g. gs://my-dir/temp>.')
+        'No options provided. To run with default class (WordCountIT), must'
+        ' provide --beam_it_options=--tempRoot=<temp dir,'
+        ' e.g. gs://my-dir/temp>.')
   if FLAGS.beam_sdk is None:
     raise errors.Config.InvalidValue(
         'No sdk provided. To run Beam integration benchmark, the test must'
-        'specify which sdk is used in the pipeline. For example, java/python.'
-    )
-  if benchmark_spec.dpb_service.service_type != dpb_service.DATAFLOW:
+        'specify which sdk is used in the pipeline. For example, java/python.')
+  if benchmark_config_spec.dpb_service.service_type != dpb_service.DATAFLOW:
     raise NotImplementedError('Currently only works against Dataflow.')
+  if (len(FLAGS.beam_it_options) > 0 and
+      (not FLAGS.beam_it_options.endswith(']') or
+       not FLAGS.beam_it_options.startswith('['))):
+    raise Exception("beam_it_options must be of form"
```

Comment:
I'm not entirely sure what you mean. If there is a correct exception type to be using here, I'd definitely be excited to switch to using it.